pegasus/sites.v3/hourofcode.com/public/how-to/parents.md

```diff
@@ -0,0 +1,77 @@
+---
+title: <%= hoc_s(:title_how_to_parents) %>
+layout: wide
+nav: how_to_nav
+---
+
+<%= view :signup_button %>
+
+# How to complete an Hour of Code with your child
+Complete an Hour of Code at home, or volunteer to bring it to your child’s classroom! Here’s how:
+
+## 1) Watch this how-to video
+<iframe width="500" height="255" src="//www.youtube.com/embed/SrnvvWDm73k" frameborder="0" allowfullscreen></iframe>
+
+## 2) Choose a tutorial:
+We provide a variety of fun, hour-long tutorials for students of all ages, created by a variety of partners. [Try them out!](<%= resolve_url('https://code.org/learn') %>)
+
+**All Hour of Code tutorials are self-guided - allowing your child to work at their own pace and skill-level**
+
+[<img src="/images/fit-700/tutorials.png">](<%= resolve_url('https://code.org/learn') %>)
+
+## 3) Plan your technology needs
+The best Hour of Code experience will be with Internet-connected computers. But you **don’t** need a computer, and you can even do the Hour of Code without a computer at all.
```

Comment:
Isn't not needing a computer and doing it without a computer kind of the same thing?